src/course-outline/card-header/CardHeader.scss

```
@@ -15,6 +15,7 @@
   .item-card-edit-icon {
     opacity: 0;
     transition: opacity .3s linear;
+    margin-right: 5px;
```

For symmetry on both sides of the status badge
```suggestion
    margin-right: 0.5rem;
```